app/controllers/suggestion.rb

```diff
@@ -0,0 +1,77 @@
+get '/suggestions' do
+  @comments = Comment.all
+  @suggestions = Suggestion.all #define instance variable for view
+  erb :'suggestions/index' #show all suggestions view (index)
+end
+
+get '/suggestions/new' do
+  @suggestion = Suggestion.new
+  erb :'suggestions/new' #show new suggestions view
+end
+
+post '/suggestions' do
+  # user = User.find_by(id: @user)
+  # @suggestion = user.suggestions.new(params[:suggestion])
+   # binding.pry
+   @suggestion = Suggestion.new(params[:suggestion])
+   #@suggestion.user = current_user
```

Comment:
This line that you commented out is a good idea. It would prevent against the spoofing of suggestions. (Since all data sent from a form can be spoofed.) When you get login to work, use this line.